x-pack/plugins/alerts/server/alerts_client.test.ts

```diff
@@ -39,7 +43,11 @@ beforeEach(() => {
   alertsClientParams.createAPIKey.mockResolvedValue({ apiKeysEnabled: false });
   alertsClientParams.invalidateAPIKey.mockResolvedValue({
     apiKeysEnabled: true,
-    result: { error_count: 0 },
+    result: {
+      invalidated_api_keys: [],
```

Comment:
Nope - not new, it's a fix I had to make to get typing to work.
This is something that was missed by a previous PR because the typing wasn't setup correctly...